<launch>
  <node pkg="rocon_solution_server" name="sensor_service" type="s_manager.py" output="screen">
    <param name="service" value="$(find rocon_solution_server)/services/SensorCalibration.service"/>
    <param name="reviewer" value="review_sensor"/>
  </node>
</launch>
